web/views_whiteboard.py

```
@@ -0,0 +1,212 @@
+        ).exists()
+    
+    if not (is_teacher or is_enrolled):
+        messages.error(request, "You don't have access to this classroom.")
+        return redirect('virtual_classroom_list')
+    
+    # Get or create whiteboard for this classroom
+    whiteboard, created = VirtualClassroomWhiteboard.objects.get_or_create(
+        classroom=classroom,
+        defaults={
+            'canvas_data': {},
+            'last_updated_by': request.user
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)